```
I0826 07:47:28.048] +++ [0826 07:43:15] linux/amd64: go build started
I0826 07:47:28.049] +++ [0826 07:47:14] linux/amd64: go build finished
I0826 07:47:28.049] +++ [0826 07:43:15] darwin/amd64: go build started
I0826 07:47:28.049] # k8s.io/kubernetes/pkg/volume/local
I0826 07:47:28.049] pkg/volume/local/local.go:235: undefined: volume.IsSameFSGroup
I0826 07:47:28.050] !!! [0826 07:46:30] Call tree:
I0826 07:47:28.050] !!! [0826 07:46:30]  1: /go/src/k8s.io/kubernetes/hack/lib/golang.sh:696 kube::golang::build_binaries_for_platform(...)
I0826 07:47:28.050] !!! [0826 07:46:30]  2: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
I0826 07:47:28.050] +++ [0826 07:43:15] windows/amd64: go build started
I0826 07:47:28.050] +++ [0826 07:47:28] windows/amd64: go build finished
I0826 07:47:28.059] Makefile:92: recipe for target 'all' failed
I0826 07:47:28.062] Makefile:419: recipe for target 'cross' failed
```

--- a/pkg/volume/local/local.go
+++ b/pkg/volume/local/local.go
@@ -20,7 +20,6 @@ import (
 	"fmt"
 	"github.com/golang/glog"
 	"os"
-	"syscall"
 
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
@@ -233,7 +232,7 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 
 		if len(refs) > 0 {
 			fsGroupNew := int64(*fsGroup)
-			fsGroupSame, fsGroupOld, err := isSameFSGroup(m.globalPath, fsGroupNew)
+			fsGroupSame, fsGroupOld, err := volume.IsSameFSGroup(m.globalPath, fsGroupNew)
 			if err != nil {
 				err = fmt.Errorf("failed to check fsGroup for %s (%v)", m.globalPath, err)
 				return err
@@ -291,20 +290,6 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		}
 	}
 	return nil
-}
-
-// isSameFSGroup is called only for requests to mount an already mounted
-// volume. It checks if fsGroup of new mount request is the same or not.
-// It returns false if it not the same. It also returns current Gid of a path
-// provided for dir variable.
-func isSameFSGroup(dir string, fsGroup int64) (bool, int, error) {
-	info, err := os.Stat(dir)
-	if err != nil {
-		glog.Errorf("Error getting stats for %s (%v)", dir, err)
-		return false, 0, err
-	}
-	s := info.Sys().(*syscall.Stat_t)
-	return int(s.Gid) == int(fsGroup), int(s.Gid), nil
 }
 
 type localVolumeUnmounter struct {

--- a/pkg/volume/volume_linux.go
+++ b/pkg/volume/volume_linux.go
@@ -89,3 +89,17 @@ func SetVolumeOwnership(mounter Mounter, fsGroup *int64) error {
 		return nil
 	})
 }
+
+// IsSameFSGroup is called only for requests to mount an already mounted
+// volume. It checks if fsGroup of new mount request is the same or not.
+// It returns false if it not the same. It also returns current Gid of a path
+// provided for dir variable.
+func IsSameFSGroup(dir string, fsGroup int64) (bool, int, error) {
+	info, err := os.Stat(dir)
+	if err != nil {
+		glog.Errorf("Error getting stats for %s (%v)", dir, err)
+		return false, 0, err
+	}
+	s := info.Sys().(*syscall.Stat_t)
+	return int(s.Gid) == int(fsGroup), int(s.Gid), nil
+}

--- a/pkg/volume/volume_unsupported.go
+++ b/pkg/volume/volume_unsupported.go
@@ -21,3 +21,7 @@ package volume
 func SetVolumeOwnership(mounter Mounter, fsGroup *int64) error {
 	return nil
 }
+
+func IsSameFSGroup(dir string, fsGroup int64) (bool, int, error) {
+	return true, int(fsGroup), nil
+}

